fl4health/clients/basic_client.py

```diff
@@ -360,6 +361,18 @@ def set_optimizer(self, config: Config) -> None:
         assert not isinstance(optimizer, dict)
         self.optimizer = optimizer
 
+    def clone_model(self, model: nn.Module) -> nn.Module:
```

Comment:
This does a bit more than clone the model. It also makes it "untrainable." So I might suggest calling this `clone_and_freeze_model` or something similar.

For method comments, we're trying to use the Google Docstring format. VS Code will autogenerate the structure for you if you follow the autoDocstring installation instructions in the README.MD